README.rst

```diff
@@ -51,6 +51,32 @@ Getting started
 
         result = await conn.execute(users.select(users.c.name.startswith('D')))
         d_users = await result.fetchall()
+        #
```

Comment:
Can you remove these commented examples. I might add more examples later but we can leave them out of this PR.